docs/src/components/mdx.js

```diff
@@ -305,7 +305,7 @@ const AnchorIcon = styled(SvgAnchor)`
   height: 100%;
 `;
 
-const Header = tag => ({ id, children }) => {
+const Header = ({ tag, id, children }) => {
```

Comment:
There is not a great way to assign propTypes to a curried component so I flattened it and in the component map, spread the tag and incoming props.